scripts/partition_manager/partition_manager.rst

```diff
@@ -47,39 +50,39 @@ This file must be stored in the same folder as the main :file:`CMakeLists.txt` f
    The root application does not need to define a :file:`pm.yml` file, because its partition size and placement is implied by the size and placement of the child image partitions.
    If a root application defines a :file:`pm.yml` file, it is silently ignored.
 
-Partition Manager configuration can be also provided by a subsystem.
-Subsystem Partition Manager configurations cannot define image partitions.
+Subsystems can also provide Partition Manager configuration files.
+(Here, *subsystem* means a software component, like support for a file system.)
+Subsystems cannot define image partitions.
 
-There are some limitations when multiple images include the same subsystem which defines a Partition Manager configuration.
-All partitions are global, and will only be included once, even if multiple configurations define them.
-When multiple configurations define the same partition, the configuration of that partition must be identical in all definitions.
-An exception is raised if the configuration differs.
-
-The listing below shows how to properly define the Partition Manager configuration for a subsystem.
-This must be placed in the :file:`CMakeLists.txt` that defines the subsystem.
+To add a Partition Manager configuration file for a subsystem, place the following line in its :file:`CMakeLists.txt` file (this is only required for subsystems, not applications):
 
 .. code-block:: cmake
 
   add_partition_manager_config(pm.yml)
 
+There are some limitations when multiple application images build the same subsystem code if it adds a Partition Manager configuration file in this way.
+In particular, partition definitions are global and must be identical across calls to ``add_partition_manager_config()``.
+If the same partition is defined twice with different configurations, Partition Manager will fail.
```

Comment:
It might not be clear to the reader which sub-systems that are defining a partition manager conf file.
And thus, it might be hard to know how to avoid making mistakes.

Should we reference the subsys that currently defines a `pm.yml` ?
Currently I find `subsys/settings/` and `subsys/fs` having a `pm.yml`

Reply:
To me it is sufficient to name the function required. A simple search for this in your repo will show all subsystems which has a pm.yml. A list of subsystems that define pm.yml sounds like something that will quickly be incorrect as people adding pm.ymls to subsystems forget to update this list.

Reply:
I agree with @hakonfam